l10n_th_tin_data/models/res_partner.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2020 Poonlap V. <poonlap@tanabutr.co.th>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import logging
+import pprint
+import re
+from collections import OrderedDict
+from logging import INFO
+from os import path
+from pathlib import Path
+
+import requests
+from requests import Session
+from zeep import Client, Transport, helpers
+
+from odoo import _, api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    branch = fields.Char(
+        string="Tax Branch", help="Branch ID, e.g., 0000, 0001, ...", default="00000"
+    )
```

Comment:
```suggestion
```
I see `l10n_th_partner` already has this field.

Reply:
Actually, I just want to set the default Branch number to '00000'. So, I refine the field in l10n_th_partner. 
Is there any better way to do? Please advise.

Reply:
Use ir.default ?